src/CHANGES.txt

```diff
@@ -96,6 +95,10 @@ RELEASE 3.1.0.alpha.yyyymmdd - NEW DATE WILL BE INSERTED HERE
       Specifically, this fixes the test cases use by Configure.CheckCC() which
       would fail when using -Wstrict-prototypes.
 
+  From Paweł Tomulik:
+    - Fixed must_contain(), must_not_contain(), etc. to work with substrings
```

Comment:
Change to note that this is part of the testing framework and also what class/package it belongs to?